docs/cli.md

```diff
@@ -4,22 +4,70 @@ This page describes how to setup and use the QMK CLI.
 
 # Overview
 
-The QMK CLI makes building and working with QMK keyboards easier. We have provided a number of commands to help you work with QMK:
+The QMK CLI makes building and working with QMK keyboards easier. We have provided a number of commands to make working with QMK easier and more streamlined.
```

Comment:
How about mentioning a couple of examples?

```suggestion
The QMK CLI makes building and working with QMK keyboards easier. We have provided a number of commands to simplify and streamline tasks such as obtaining and compiling the QMK firmware, creating keymaps, and more.
```

Otherwise, it's not until you scroll down to the list of subcommands that you really get an idea of what the CLI is good for.